rootfs/bin/get_object

```diff
@@ -2,12 +2,10 @@
 
 GET_PATH=/tmp/slug.tgz
 
-export BUCKET_FILE=/var/run/secrets/deis/objectstore/creds/builder-bucket
 if [ "$BUILDER_STORAGE" == "minio" ]; then
   mkdir -p /app/objectstore/minio/
   echo "git" > /app/objectstore/minio/builder-bucket
```

Comment:
@kmala hey, do you know if I can move this to the `normalize_storage` script as well? It seems only needed when 'getting' the object, but I'm not sure what it is doing. Thanks!